client/src/app/index.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
```

Comment:
It's not important notes but you can rename BrowserRoute if you want like this
```suggestion
import { BrowserRouter as Router , Route, Switch } from 'react-router-dom';
```
and then you can use it
```jsx
<Router>
        <Switch>
          <Route exact path={ROUTES.COHORT_PAGE} component={AdminCohrtPage} />
        </Switch>
      </Router>
```